client/blocks/comments/post-comment-list.jsx

```diff
@@ -363,6 +423,61 @@ class PostCommentList extends Component {
 
 	handleFilterClick = ( commentsFilter ) => () => this.props.onFilterChange( commentsFilter );
 
+	getDisplayedCollapsedInlineComments = ( commentsTree ) => {
+		// Only take the most recent comment.
+		const lastCommentArr = commentsTree.children.slice( -1 );
+		const lastComment = lastCommentArr[ 0 ];
+
+		if ( ! lastComment ) {
+			return {
+				displayedComments: [],
+				displayedCommentsCount: 0,
+				commentsTreeToUse: commentsTree,
+			};
+		}
+
+		// Setup a new comment tree to customize replies rendered.
+		const newCommentTree = { children: lastCommentArr };
+		newCommentTree[ lastComment ] = {
+			data: commentsTree[ lastComment ]?.data,
+			children: [],
+		};
+
+		// Go through the children of the last comment to find replies by the current user.
+		const authorReplies = commentsTree[ lastComment ]?.children.filter( ( replyId ) => {
```

Comment:
I think it would be better to search all comments for an author reply, not just the latest thread. I also think it would be better to show the comment thread with the latest reply rather than the latest top level comment.

Having said that I think that's just a nit pick/ opinion and it works well as is.